glossary/glossary.md

```diff
@@ -0,0 +1,57 @@
+# Multimedia 
++ #### *Packet* is a formatted unit of data transmitted over network 
++ #### *Frame* is a basic data unit used by media coding formats
++ #### (media)*Track* is equivalent to a single audio or video 
++ #### Web protocols:
+  + #### *UDP*(User Datagram Protocol) is a [transport layer](https://en.wikipedia.org/wiki/OSI_model#Layer_4:_Transport_layer) protocol using connectionless communication
+  + #### *TCP*(Transmission Control Protocol) is a [transport layer](https://en.wikipedia.org/wiki/OSI_model#Layer_4:_Transport_layer) protocol using connection-oriented communication
+  + #### *RTP*(Real-time Transport Protocol) is a network protocol for delivering audio and video over IP networks
+  + #### *HTTP*(Hypertext Transfer Protocol) is a protocol for fetching data from a server by a client
```

Comment:
I would prefer some more precise defintion (perhaps it is good to mention where it's position the ISO-OSI layer?) along with an information, where HTTP is used in multimedia processing (some reference to HLS and MPEG DASH)